src/cryptoadvance/specter/static/style.css

```diff
@@ -189,6 +194,24 @@
   .animate-spin-slow {
     animation: spin 1.5s linear infinite;
   }
+
+  /*
+   *
+   * ScrollBars
+   *
+   */
+
+  *::-webkit-scrollbar-track {
```

Comment:
Still looks like this on Chrome for me:
<img width="311" alt="grafik" src="https://user-images.githubusercontent.com/70536101/216669540-c4c2f11d-1ada-4e19-b906-f160343141a7.png">

and

<img width="311" alt="grafik" src="https://user-images.githubusercontent.com/70536101/216669629-5ace827b-a76f-4424-bc58-5aa47ab0e788.png">

Reply:
Ah wait, I am on your old branch. Wait a sec.

Reply:
This is very odd, i tested it on mine and it works with the new styles, on both Chrome & Arc.

Reply:
I am on: Version 109.0.5414.119 (Official Build) (arm64)